src/containers/dashboard/DashboardContainer.jsx

```diff
@@ -11,22 +11,29 @@ type Props = {};
 
 type State = {
 	serverState: ServerState,
+	sensorConnectionState: ConnectionState,
 	activeMeasurement: ?Measurement
 };
 
 export default class DashboardContainer extends React.Component<Props, State> {
 	isUnmounted = false;
 	source: CancelTokenSource = CancelToken.source();
-	timeout: TimeoutID;
```

Comment:
`setTimeout` was on purpose. `setInterval` might cause problems with the AJAX calls.